hoot-core-test/src/test/cpp/hoot/core/ops/MapCropperTest.cpp

```diff
@@ -257,7 +257,7 @@ class MapCropperTest : public HootTestFixture
     //compare relations
     const RelationMap relations = map->getRelations();
     HOOT_STR_EQUALS(1, relations.size());
-    QString relationStr = "relation(-1592); type: multipolygon; members:   Entry: role: outer, eid: Way(-1556);   Entry: role: inner, eid: Way(-1552); ; tags: landuse = farmland; status: invalid; version: 0; visible: 1";
+    QString relationStr = "relation(-1592); type: multipolygon; members:   Entry: role: outer, eid: Way(-1556);   Entry: role: inner, eid: Way(-1552); ; tags: landuse = farmland; status: invalid; version: 0; visible: 1; circular error: 15";
     for (RelationMap::const_iterator it = relations.begin(); it != relations.end(); ++it)
```

Comment:
This output change is due to the sorting now being done on the output.  The sorting was necessary for consistent output when running single or multithreaded.